htdocs/postdeploy.php

```diff
@@ -42,8 +42,10 @@
 $sqls         = file_get_contents($path_to_file);
 $conn->exec($sqls);
 
+$pw = password_hash($password, PASSWORD_DEFAULT);
+
 $conn->query(
-    "UPDATE users SET Password_MD5=CONCAT('aa', MD5('aa$password')), 
+    "UPDATE users SET Password_hash=$pw,
```

Comment:
`$pw` needs to be database quoted with $conn->quote.. (even better would be a prepared statement, but as it's written it'll just be invalid SQL..)